### PR TITLE
Support (automatic) operator upgrades with OLM

### PR DESCRIPTION
**Pull request by razvan**
Fixes https://github.com/stackabletech/issues/issues/571

### CODE REVIEW

File: olm/multi-op-catalog.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+fi
+
+rm -rf olm/catalog/${OPERATOR}
+rm -rf olm/catalog.Dockerfile
+
+mkdir -p olm/catalog/${OPERATOR}
+opm generate dockerfile olm/catalog
+
+opm init "stackable-${OPERATOR}-operator" \
+	--default-channel=stable \
```
Comment (siegfriedweber) [inline olm/multi-op-catalog.sh:50]: Just to replicate the current state:

```suggestion
	--default-channel=24.4 \
```

File: olm/multi-op-catalog.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	echo "- name: ${OPERATOR}-operator.v24.3.0-1"
+	echo "  replaces: ${OPERATOR}-operator.v23.11.0"
+	echo "  skips:"
+	echo "  -  ${OPERATOR}-operator.v24.3.0"
+} >>"olm/catalog/${OPERATOR}/stackable-${OPERATOR}-operator.yaml"
+
+echo "Render operator: ${OPERATOR}"
+opm render "docker.stackable.tech/sandbox/${OPERATOR}-bundle:23.11.0" --output=yaml >>"olm/catalog/${OPERATOR}/stackable-${OPERATOR}.v23.11.0-bundle.yaml"
+opm render "docker.stackable.tech/sandbox/${OPERATOR}-bundle:24.3.0" --output=yaml >>"olm/catalog/${OPERATOR}/stackable-${OPERATOR}-v24.3.0-bundle.yaml"
+opm render "docker.stackable.tech/sandbox/${OPERATOR}-bundle:24.3.0-1" --output=yaml >>"olm/catalog/${OPERATOR}/stackable-${OPERATOR}-v24.3.0-1-bundle.yaml"
```
Comment (siegfriedweber) [inline olm/multi-op-catalog.sh:72]:
```suggestion
opm render "docker.stackable.tech/sandbox/${OPERATOR}-bundle:24.4.0-1" --output=yaml >>"olm/catalog/${OPERATOR}/stackable-${OPERATOR}-v24.4.0-1-bundle.yaml"
```

**Review by siegfriedweber — CHANGES_REQUESTED** [review]

File: olm/build-manifests.py
[the remark this answers is not in the document]
Comment (razvan) [reply olm/build-manifests.py:1]: Have added a comment: https://github.com/stackabletech/stackable-utils/pull/67/commits/75a7effeb79f99d5a4c061a8891f48c3a3cad63d

**Review by siegfriedweber — APPROVED** [review]
LGTM

**Pull request merged**